clients/js/test/client.test.ts

```diff
@@ -196,4 +196,38 @@ test('wrong code returns an error', async () => {
     const results = await collection.get(undefined, { "test": { "$contains": "hello" } });
     expect(results.error).toBeDefined()
     expect(results.error).toBe("ValueError('Expected one of $gt, $lt, $gte, $lte, $ne, $eq, got $contains')")
-})
\ No newline at end of file
+})
+
+test('it should return an error when inserting duplicate IDs', async () => {
```

Comment:
let's fix the description of this test to describe what it is actually testing 

suggestion: it should return an error when a duplicate ID is inserted